docs/en/compute-edition/22-12/admin-guide/install/system-requirements.adoc

```diff
@@ -50,24 +50,26 @@ endif::compute_edition[]
 
 Each Defender requires 256MB of RAM and 8GB of host storage.
 
-The Defender uses `cgroups` to cap resource usage at 512MB of RAM and 900 CPU shares where a typical load is ~1-5% CPU and 30-70MB RAM.
+The Defender uses `cgroups` v1 or v2 to cap resource usage at 512MB of RAM and 900 CPU shares where a typical load is ~1-5% CPU and 30-70MB RAM.
 
-The Defender stores its data in the `/var` folder.
-When allocating disk space for Defender, ensure the required space is available in the `/var` folder.
-Defenders are designed to be portable containers that collect data.
-Any data that must be persisted is sent to the Prisma Cloud Console for storage.
-Defenders don't require persistent storage.
-If you deploy persistent storage for Defenders, it can corrupt Defender files.
+For more than 10,000 Defenders you need 4 vCPUS and 10GB of RAM for every additional 5,000 Defenders For example, 20,000 connected Defenders require a total of 16 vCPUs, 50GB of RAM and 500GB SSD of persistent storage.
 
 If Defenders provide registry scanning they require the following resources:
 
-* Defenders providing registry scanning--
 * 2GB of RAM
 * 20GB of storage
 * 2 CPU cores
-Defenders that are part of CI integrations (Jenkins, twistcli) require storage space depending on the size of the scanned images.
-The required disk space is 1.5 times the size of the largest image to be scanned, per executor.
-For example, if you have a Jenkins instance with two executors, and your largest container image is 500MB, then you need at least 1.5GB of storage space: `500MB x 1.5 x 2`
+
+For Defenders that are part of CI integrations (Jenkins, twistcli), storage space depends on the size of the scanned images. The required disk space is 1.5 times the size of the largest image to be scanned, per executor. For example, if you have a Jenkins instance with two executors, and your largest container image is 500MB, then you need at least 1.5GB of storage space: `500MB x 1.5 x 2`.
+
+Keep the following caveats in mind when determining Defender storage and memory requirements:
+
+* The Defender stores its data in the `/var` folder. When allocating disk space for Defender, ensure the required space is available in the `/var` folder.
+* Defenders are designed to be portable containers that collect data. Any data that must be persisted is sent to the Prisma Cloud Console for storage.
+* Defenders don't require persistent storage. If you deploy persistent storage for Defenders, it can corrupt Defender files.
```

Comment:
Defenders do not require persistent storage. Deploying persistent storage for Defenders can corrupt Defender files.